ethclient/ethclient.go

```diff
@@ -278,6 +278,14 @@ func (ec *Client) TransactionReceipt(ctx context.Context, txHash common.Hash) (*
 	return r, err
 }
 
+// BlockReceipts returns the receipts of a transaction by block number.
+func (ec *Client) BlockReceipts(ctx context.Context, number *big.Int) (types.Receipts, error) {
+	var result types.Receipts
+	err := ec.c.CallContext(ctx, &result, "eth_getBlockReceipts", toBlockNumArg(number))
```

Comment:
It seems that both` web3ext.go` and `web3.js` need to be modified to bind `"eth_getBlockReceipts"` and `"GetBlockReceipts"`, and expose `"eth_getBlockReceipts"`.